docs/05-e2e-testing.md

```
@@ -0,0 +1,118 @@
+
+```bash
+docker run -p 8080:8080 -e KC_BOOTSTRAP_ADMIN_USERNAME=admin -e KC_BOOTSTRAP_ADMIN_PASSWORD=admin quay.io/keycloak/keycloak:26.1.4 start-dev
+```
+
+Access Keycloak Admin Console:
+```
+http://your-server-ip:8080
+```
+
+Log in using admin credentials:
+- Username: `admin`
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)